### ISSUE TITLE
Implement interface for MS SQL server

### ISSUE THREAD

**Issue by melvin15may** [dssg/pgdedupe#39]
It will be great if we can interface with MS SQL server. The interaction with sql server is different from postgres. We can have a new attribute in the configuration to specify the type of database (default being postgres).

**Comment by mbauman** [comment]
I absolutely agree!  Unfortunately, I don't have bandwidth to accomplish that myself at this time, but I'd welcome a pull request.  This is forked from dedupeio/dedupe-examples, where there is a MySQL example, but no MS SQL.

**Comment by melvinmathew** [comment]
I am currently working on implementing the MySQL dedupe-example in MS SQL, so I can work on this in parallel.

**Comment by melvin15may** [comment]
I am currently working on implementing the MySQL dedupe-example in MS SQL, so I can work on this in parallel.

**Comment by mbauman** [comment]
We have decided that this is out-of-scope for this repository, which is explicitly limited to a Postgres implementation (since that's what we use and will support).
